app/decorators/news_article_decorator.rb

```diff
@@ -6,4 +6,16 @@ def related_sections
       related_section == object.section
     end
   end
+
+  def publishers
```

Comment:
This method creates confusion with `NewsArticle.publisher` - is this field no longer required if we instead use the distribution list?